bay/images/Dockerfile.ci-builder

```diff
@@ -1,40 +1,64 @@
 FROM hashicorp/terraform:light AS terraform
-FROM integratedexperts/ci-builder:latest
+FROM uselagoon/php-7.4-cli:latest
```

Comment:
You could probably just replace this with 

```suggestion
FROM php:7.4-cli
```

Reply:
This would ditch all the goofy ssh key shenanigans

Reply:
👍  I'll give that a go.